data/data/alibabacloud/bootstrap/outputs.tf

```diff
@@ -1,3 +1,13 @@
 output "bootstrap_ip" {
   value = local.is_external ? data.alicloud_instances.bootstrap_data.instances.0.public_ip : data.alicloud_instances.bootstrap_data.instances.0.private_ip
 }
+output "slb_ids" {
+  value = var.slb_ids
+}
+output "master_ecs_ids" {
+  value = var.master_ecs_ids
+}
```

Comment:
These should not be outputs from the bootstrap stage.

Reply:
I tried a few different iterations but received errors about the variables not being set. I'll try removing them from the outputs.

Reply:
Removed.